src/rust/ide/enso-protocol/src/file_manager.rs

```diff
@@ -9,9 +9,13 @@
 #![warn(missing_copy_implementations)]
 #![warn(missing_debug_implementations)]
 
-use crate::prelude::*;
+use enso_prelude::*;
```

Comment:
Why not have crate-specific prelude?

Reply:
Funny enough, the original code was:

```rust
use crate::prelude::*;
use enso_prelude as prelude;
```

But it doesn't compile anymore and I didn't want to lose time investigating it.

Reply:
omg, that's what happens when you work overnight. Thanks for pointing the obvious. I will fix it.